packages/lib/modules/web3/contracts/useMulticall.ts

```diff
@@ -33,6 +36,9 @@ export function useMulticall(multicallRequests: ChainContractConfig[], options:
           const results = await multicall(config, {
             contracts: multicalls,
             chainId: Number(chain),
+            ...(Number(chain) === getChainId(GqlChain.Sonic) && {
```

Comment:
Created a task to not forget about removing this: 
https://github.com/orgs/balancer/projects/12/views/1?pane=issue&itemId=92621002&issue=balancer%7Cfrontend-monorepo%7C399